modules/gcp_bigquery_masked_dataset/main.tf

```diff
@@ -0,0 +1,66 @@
+resource "random_id" "random_id" {
+  byte_length = 4
+}
+
+resource "google_kms_key_ring" "google_kms_key_ring" {
+  name     = "${var.name}_${random_id.random_id.hex}"
+  location = var.location
+}
+
+resource "google_kms_crypto_key" "google_kms_crypto_key" {
+  #checkov:skip=CKV_GCP_82:Need to be able to delete because of integration test
+  name            = "${var.name}_${random_id.random_id.hex}"
+  key_ring        = google_kms_key_ring.google_kms_key_ring.id
+  rotation_period = "7776000s"
```

Comment:
If this does not support other units, please either use a local doing the multiplication for this, or at least include a comment.